addons/mass_mailing/views/snippets_themes.xml

```diff
@@ -851,14 +851,6 @@
         data-drop-near="[data-oe-field='body_html']:not(:has(.o_layout)) > *, .oe_structure > *"
         data-drop-in="[data-oe-field='body_html']:not(:has(.o_layout)), .oe_structure"/>
 
-    <div data-js="SnippetSave"
-        t-att-data-selector="mailing_content_selector">
-        <we-button class="fa fa-fw fa-save"
-                   title="Save the block to use it elsewhere"
-                   data-save-snippet=""
-                   data-no-preview="true"/>
-    </div>
```

Comment:
Tricky case. This is the right fix for master but this is could be more stable. You are removing something people may base xpath on, so potentially breaking custo. For 14.0, I would add a comment above this block "TODO remove in master, the option has been disabled in 14.0 because of tricky problems to resolve that require refactoring" and you can virtually disable it by including the `SnippetSave` class in the mass mailing editor JS file. This also is better for stable version as people on the Saas would not receive your fix here: it requires a module update which we don't do, so they would be stuck with the bug.